MenuExample/Controller/ViewController.swift

```diff
@@ -24,6 +28,14 @@ class ViewController: UIViewController {
                 super.prepareForSegue(segue, sender: sender)
         }
     }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        menuAnimator = MenuTransitionAnimator(shouldPassEvents: false, tappedOutsideHandler: {
```

Comment:
@EgorSobko does it make sense to rename 'shouldPassEvents' to 'shouldPassEventsOutsideMenu' ?